core/src/mast/node/basic_block_node/mod.rs

```diff
@@ -224,6 +224,12 @@ impl BasicBlockNode {
             decorator_ids.into_iter().map(|decorator_id| (after_last_op_idx, decorator_id)),
         );
     }
+
+    /// Used to initialize decorators for the [`BasicBlockNode`]. Replaces the existing decorators
+    /// with the given ['DecoratorList'].
+    pub fn init_decorators(&mut self, decorator_list: DecoratorList) {
```

Comment:
```suggestion
    pub fn set_decorators(&mut self, decorator_list: DecoratorList) {
```
Nit: `set` instead of `init` seems more idiomatic.